AWSCoreTests/AWSTestUtility.m

```diff
@@ -65,6 +65,21 @@ + (void)initialize {
     [super initialize];
 }
 
++ (void) setupSTSBasedSessionCredentialsProvider {
+    NSString *filePath = [[NSBundle bundleForClass:[self class]] pathForResource:@"credentials"
+                                                                          ofType:@"json"];
+    NSDictionary *credentialsJson = [NSJSONSerialization JSONObjectWithData:[NSData dataWithContentsOfFile:filePath]
+                                                                    options:NSJSONReadingMutableContainers
+                                                                      error:nil];
+    AWSBasicSessionCredentialsProvider *credentialsProvider = [[AWSBasicSessionCredentialsProvider alloc]
+                                                               initWithAccessKey:credentialsJson[@"accessKey"]
+                                                                       secretKey:credentialsJson[@"secretKey"]
+                                                                    sessionToken:credentialsJson[@"sessionToken"]];
+    AWSServiceConfiguration *configuration = [[AWSServiceConfiguration alloc] initWithRegion:AWSRegionUSEast1
```

Comment:
Read this from the config rather than hardcoding it

Reply:
done